packages/handlersjs-http/lib/handlers/http-cors-request.handler.spec.ts

```diff
@@ -56,14 +33,26 @@ describe('HttpCorsRequestHandler', () => {
         url: new URL('http://example.com'),
         method: 'GET',
         headers: {
-          accEPt: 'text/plain',
+          accept: 'text/plain',
           origin: 'http://test.de',
         },
       },
     };
 
   });
 
+  it('should be correctly instantiated', () => {
+
+    expect(new HttpCorsRequestHandler(handler)).toBeTruthy();
+    expect(new HttpCorsRequestHandler(handler, undefined)).toBeTruthy();
+    expect(new HttpCorsRequestHandler(handler, mockOptions)).toBeTruthy();
+
+    // cover the optional abstract parameters
+    class t extends HttpCorsOptions {}
+    new t();
```

Comment:
Not sure what this is for ...

Reply:
This was to cover line 10-15 (the optional constructor parameters for the abstract HttpCorsOptions class which weren't covered yet).